cas-server-support-oidc/src/test/java/org/apereo/cas/services/OidcRegisteredServiceTests.java

```diff
@@ -0,0 +1,49 @@
+package org.apereo.cas.services;
+
+import org.apache.commons.io.FileUtils;
+import org.apereo.cas.services.JsonServiceRegistryDao;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redundant import from the same package - org.apereo.cas.services.JsonServiceRegistryDao.](https://www.codacy.com/app/mmoayyed/cas/file/2530481895/issues/source?bid=3256115&fileBranchId=3273836#l4)